src/acquisition/covidcast/database.py

```diff
@@ -106,6 +106,7 @@ def insert_or_update_batch(self, cc_rows, batch_size=2**20, commit_partial=False
 
     tmp_table_name = 'tmp_insert_update_table'
 
+    # TODO: this heavily copypastas src/ddl/covidcast.sql -- theres got to be a better way
```

Comment:
Is it worth creating an issue for tracking this or do  you typically leave as todo?

Reply:
Leaving a TODO essentially creates a lowest-priority task. If it becomes more obviously important we can escalate to an issue.